whisper_tests/src/lib.rs

```diff
@@ -0,0 +1,37 @@
+extern crate rand;
+
+use std::fs::*;
+use std::path::PathBuf;
+use rand::*;
+
+#[derive(Debug)]
+pub struct CleanTempDir {
+    base_path: String,
+    files: Vec<String>,
+}
+
+impl CleanTempDir {
```

Comment:
fyi,
https://docs.rs/tempfile/3.0.2/tempfile/struct.TempDir.html

Reply:
I saw temp file, but didnt noticed about dir there. Thx, will give it a try!